cmd/influxd/server_integration_test.go

```diff
@@ -1456,9 +1456,7 @@ func TestSingleServerDiags(t *testing.T) {
 		t.Skip(fmt.Sprintf("skipping '%s'", testName))
 	}
 	dir := tempfile()
-	defer func() {
```

Comment:
I think `go lint` will complain about these changes. Not sure, but I think so. We don't run lint right now, so perhaps it doesn't matter.